srcpkgs/cni-plugins/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'cni-plugins'
+pkgname=cni-plugins
+version=0.7.4
+revision=1
+wrksrc="plugins-${version}"
+build_style=go
+go_import_path="github.com/containernetworking/plugins"
+short_desc="Some standard networking plugins, maintained by the CNI team"
+maintainer="Kyle Nusbaum <knusbaum+void@sdf.org>"
+license="Apache-2.0"
+homepage="https://github.com/containernetworking/plugins"
+distfiles="https://github.com/containernetworking/plugins/archive/v${version}.tar.gz"
+checksum=d47f2bda2bfd6699843a98d91a2e5c40aa58eacd3f2393c7696c158c410aad3c
+
+do_build() {
+	./build.sh
+}
+
+do_install() {
+	vmkdir /usr/libexec/cni
+	cp bin/* ${DESTDIR}/usr/libexec/cni
```

Comment:
I might be wrong but i think vcopy can be used here

`vcopy bin/* usr/libexec/cni`